src/standalone/Bucket/artifacts/artifacts.jl

```diff
@@ -49,3 +52,51 @@ function bareground_albedo_dataset_path()
     path = AW.get_data_folder(bareground_albedo_dataset)
     return joinpath(path, "bareground_albedo.nc")
 end
+
+"""
+    era5_t2m_sp_u10n_dataset_path()
+
```

Comment:
should we note that these are used only in testing?

Reply:
should these live in `Bucket` or are they more general?

Reply:
Yes we should note that they're for testing. They're general enough to be used beyond just the bucket. What do you think about moving them to `test/`?